### ISSUE TITLE
Migrating to this version of concord4, Homebridge configuration question

### ISSUE THREAD

**Issue by Casuallynoted** [automaton82/MQTT-concord4#1]
Hello!

I've just finished setting up an MQTT server and connecting this concord4 script to it. Seems to be working properly based on there being no errors in the log. I've been looking to try to get it connected with this homebridge MQTT alarm plugin-
https://github.com/nzbullet/homebridge-homeqtt-alarm/blob/master/README.md

But I'm a big confused as to what I should enter into the various topic fields and/or if there are any sensor codes like the guide there mentions. I thought I'd ask and see if you might have any insight into this, or if there's maybe another homebridge plugin that might be more compatible with this MQTT script that could be awesome too!

**Comment by automaton82** [comment]
I don't use homebridge so I'm not overly familiar with it, but I can say from that project there's a config JSON file - https://github.com/nzbullet/homebridge-homeqtt-alarm/blob/master/example-config.json

In there it is setting topic paths for what it expects, and those are different from what this one is sending. Once you run this one you should see log messages like TX -> .../... which shows you the URL and the topic + value being sent. You need to make what this is ending match what they expect. At the moment it's just stuff like:

* /concord/zone1/open
* /concord/zone2/closed

Basically make those topics and payloads match what that one wants and it will work. I suggest you fork this for that specific homebridge library, that will be easiest for you to modify it and it shouldn't be too hard if you know what they want.

**Comment by Casuallynoted** [comment]
Do you think this looks correct? Sorry for responding so late, I finally got to checking this out.

![Screen Shot 2020-11-18 at 4 17 27 PM](https://user-images.githubusercontent.com/6272557/99589148-9e82e880-29b9-11eb-9a5b-a23fb5f2604c.png)
